orbs/pulumi.yml

```diff
@@ -90,6 +90,34 @@ commands:
           name: "pulumi stack rm << parameters.stack >>"
           command: pulumi stack rm << parameters.stack >> <<# parameters.force >>--force<</ parameters.force >> --cwd << parameters.working_directory >>
 
+  # stack output command
+  stack_output:
+    parameters:
+      stack:
+        type: string
+        description: "Name of the stack to operate on."
+      property_name:
+        type: string
+        description: "Property name of the stack output to set to the environment variable."
+      env_var:
+        type: string
+        description: "Name of the environment variable to set the output property's value to."
+      log_to_stderr:
+        type: boolean
+        description: "Log to stderr instead of to files. Default true."
+        default: true
+      working_directory:
+        type: string
+        description:
+            "The relative working directory to use, i.e. where your Pulumi.yaml is located."
+        default: "."
+    steps:
+      - run:
+          name: "pulumi stack output << parameters.property_name >> --stack << parameters.stack >>"
+          command: | 
+            OUTPUT_VALUE=$(pulumi stack output << parameters.property_name >> --stack << parameters.stack >> <<# parameters.log_to_stderr >>--logtostderr<</ parameters.log_to_stderr >> --cwd << parameters.working_directory >>)
```

Comment:
In addition to removing `--logtostderr`, we should add `--show-secrets`.

Output properties may be "secret", meaning they are encrypted -- potentially by a user-supplied key -- and so if we don't decrypt that value, it won't be of any use to other steps in the same CircleCI job :)